src/engines/witcher/witcher.cpp

```diff
@@ -212,9 +217,10 @@ void WitcherEngine::initResources(LoadProgress &progress) {
 	ResMan.registerDataBase(_target);
 
 	progress.step("Adding extra archive directories");
-	indexMandatoryDirectory("system"      , 0,  0, 2);
-	indexMandatoryDirectory("data"        , 0,  0, 3);
-	indexMandatoryDirectory("data/modules", 0, -1, 4);
+	indexMandatoryDirectory("system"        , 0,  0, 2);
+	indexMandatoryDirectory("system/scripts", 0,  0, 3);
+	indexMandatoryDirectory("data"          , 0,  0, 4);
+	indexMandatoryDirectory("data/modules"  , 0, -1, 5);
 
 	// Contains BIFs with voices for the two premium modules
 	indexOptionalDirectory("data/voices", 0, 0, 5);
```

Comment:
Increase the priority (the last number, the 5) here as well, just so there's a clear priority progression in case the same file appears in data/modules and data/voices (as unlikely as that is).